src/glmfit.jl

```diff
@@ -205,6 +207,19 @@ function updateμ!(r::GlmResp{V,D,L}) where {V<:FPVector,D<:NegativeBinomial,L<:
     end
 end
 
+function updateμ!(r::GlmResp{V,D,L}) where {V<:FPVector,D,L<:PowerLink}
+    y, η, μ, wrkres, wrkwt, dres = r.y, r.eta, r.mu, r.wrkresid, r.wrkwt, r.devresid
+    pl = r.link
+    @inbounds for i in eachindex(y, η, μ, wrkres, wrkwt, dres)
+        μi, dμdη = inverselink(L(pl.λ), η[i])
```

Comment:
```suggestion
    y, η, μ, wrkres, wrkwt, dres, l = r.y, r.eta, r.mu, r.wrkresid, r.wrkwt, r.devresid, r.link

    @inbounds for i in eachindex(y, η, μ, wrkres, wrkwt, dres)
        μi, dμdη = inverselink(L(l.λ), η[i])
```

Reply:
instead of  `y, η, μ, wrkres, wrkwt, dres, l = r.y, r.eta, r.mu, r.wrkresid, r.wrkwt, r.devresid, r.link`
I am going to change as follows:
`y, η, μ, wrkres, wrkwt, dres = r.y, r.eta, r.mu, r.wrkresid, r.wrkwt, r.devresid, l = r.link`  
and remaining

Reply:
Pushed code as per the suggestions.